tests/unit/dialog/dialog_core.js

```diff
@@ -7,11 +7,50 @@
 // TODO add teardown callback to remove dialogs
 module("dialog: core");
 
+test( "markup structure", function( assert ) {
+	expect( 7 );
+
+	var element = $( "<div>" ).dialog({
+			buttons: [ {
+				text: "Ok",
+				click: $.noop
+			} ]
+		}),
+		widget = element.dialog( "widget" ),
+		titlebar = widget.find( ".ui-dialog-titlebar" ),
+		close = titlebar.find( ".ui-button" ),
+		buttonpane = widget.find( ".ui-dialog-buttonpane" );
+
+	assert.hasClasses( widget, "ui-dialog ui-widget ui-widget-content ui-dialog-buttons" );
+	assert.hasClasses( titlebar, "ui-dialog-titlebar" );
+	assert.hasClasses( element, "ui-dialog-content ui-widget-content" );
+	assert.hasClasses( buttonpane, "ui-dialog-buttonpane ui-widget-content" );
+	assert.hasClasses( close, "ui-dialog-titlebar-close" );
+	ok( titlebar.find( ".ui-dialog-title" ).length, "Title has class ui-dialog-title" );
```

Comment:
Why is this tested differently than `close` and why does `close` use .find( ".ui-button" )`?